modules/devices/steamdeck/fde.nix

```diff
@@ -0,0 +1,105 @@
+{ config, lib, pkgs, utils, ... }:
+
+let
+  cfg = config.jovian.devices.steamdeck;
+in
+{
+  options = {
+    jovian.devices.steamdeck = {
+      fde = {
+        deckbd = {
```

Comment:
Hmmm... not sure about the `jovian.devices.steamdeck`**`.fde`**`.deckbd` namespace...

I don't have a suggestion at the exact moment, but it's less directly about FDE, but about a device-specific tool and integration to make FDE work, so maybe it should be categorized as such. :thinking:  Any thoughts?

Reply:
My reasoning was that the launch is directly tied to FDE-related password prompting services. I too can't really think of a better name - I'm open to suggestions.

Reply:
(side-note: I'll try to get this tested and verified on my system soon, and make it so the last issues are those tiny little details so it can be merged)